IFC4x3/Sections/Resource definition data schemas/Schemas/IfcGeometricConstraintResource/Types/IfcAlignmentHorizontalSegmentTypeEnum/Documentation.md

```diff
@@ -0,0 +1,59 @@
+The IfcAlignmentHorizontalSegmentTypeEnum indicates the type of a segment of a horizontal alignment segment (IfcAlignmentHorizontalSegment). Horizontal segments can be viewed from a geometric perspective and from a kinematic perspective. In recent times the kinematic perspective gained importance. The enumerations are detailed according to this development especially in modern track design.
+
+**Kinematic perspective on horizontal alignment segments**
+
+The central parameter of the kinematic perspective is lateral acceleration of the vehicle induced by change of direction while driving. In the horizontal layout this is the represented by the curvature of the segment. According to the curvature value the following categorization can be made:
+
+| Curvature | Segmenttype        | Enumeration Values |
+|:----|:------------------|:----------|
+| 0 | straight line        | LINE |
+| constant in the complete segment, <> 0 | Circular arc  | CIRCULARARC |
+| variation along the segment | Transition with linear curvature variation | CLOTHOID, CUBIC  |
+| variation along the segment | Transition with non-linear curvature variation | HELMERTCURVE, BLOSSCURVE, COSINECURVE, SINECURVE, VIENNESEBEND |
+
+
+**Geometric perspective on horizontal alignment segments**
+
+
+
+The geometric perspective denotes in the context of the business terminology related IfcAlignment documentation the traditional view. Before the availability of modern computers alignment design was performed using "traditional" drawing techniques. In the first phase of computerization this origin led to a representation in the x,y space first and a check of safety related properties in a second step. This can still be seen in regulations which have been put into effect 1980 or earlier. Of course designs which have been produced on basis of these regulations reflect the "good enough" attitude in the precision of the documentation. 
```

Comment:
Had to switch this up. 😄 
```suggestion
The traditional view is denoted by the geometric perspective in the context of the business terminology related IfcAlignment documentation. Before the availability of modern computers alignment design was performed using "traditional" drawing techniques. In the first phase of computerization this origin led to a representation in the x,y space first and a check of safety related properties in a second step. This can still be seen in regulations which have been put into effect 1980 or earlier. Of course designs which have been produced on basis of these regulations reflect the "good enough" attitude in the precision of the documentation. 
```